jobs/payment-jobs/tasks/statement_task.py

```diff
@@ -33,36 +33,43 @@ class StatementTask:  # pylint:disable=too-few-public-methods
     """Task to generate statements."""
 
     has_date_override: bool = False
+    has_account_override: bool = False
 
     @classmethod
-    def generate_statements(cls, date_override=None):
+    def generate_statements(cls, arguments=None):
         """Generate statements.
 
         Steps:
         1. Get all payment accounts and it's active statement settings.
         """
+        date_override = arguments[0] if arguments and len(arguments) > 0 else None
+        auth_account_override = arguments[1] if arguments and len(arguments) > 1 else None
```

Comment:
allow a specific account number to be filtered on. This allows us to generate a monthly statement using the interim statement settings before the month is up without affecting other accounts